pkgs/development/libraries/onnxruntime/default.nix

```
@@ -0,0 +1,107 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, fetchpatch
+, runCommand
+, patchutils
+, pkgconfig
```

This alias was dropped
```suggestion
, pkg-config
```